src/mon/OSDMonitor.cc

```diff
@@ -93,6 +95,164 @@ const uint32_t MAX_POOL_APPLICATION_LENGTH = 128;
 
 } // anonymous namespace
 
+
+namespace {
+
+enum class flag_result_t : int { not_verified = -EPERM, not_set = 0, ok = 1 };
```

Comment:
I've had an almost complete change of heart, and wish I' written:
"enum struct". 

The /feature/ is "scoped enums", not "class enumerations".